packages/terra-consumer-nav/src/components/nav-help/NavHelp.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Arrange from 'terra-arrange';
+import ResponsiveElement from 'terra-responsive-element';
+import NavHelpContent from './NavHelpContent';
+import IconInfo from '../../icons/IconInfo';
+import Modal from '../modal/Modal';
+import Popup from '../popup/Popup';
```

Comment:
For now replaced Popup with TerraPopup where Popup from terra-core is used. Based on the list of future components terra-consumer will have its own Popup. We can replace TerraPopup with that new one and remove the current custom Popup with a custom header.